js/modules/k6/crypto/crypto.go

```diff
@@ -56,16 +58,19 @@ func New() *Crypto {
 	return &Crypto{}
 }
 
-func (*Crypto) RandomBytes(ctx context.Context, size int) []byte {
+// RandomBytes returns an ArrayBuffer with random data of given size.
+func (*Crypto) RandomBytes(ctx context.Context, size int) *goja.ArrayBuffer {
+	rt := common.GetRuntime(ctx)
 	if size < 1 {
-		common.Throw(common.GetRuntime(ctx), errors.New("invalid size"))
+		common.Throw(rt, errors.New("invalid size"))
 	}
 	bytes := make([]byte, size)
 	_, err := rand.Read(bytes)
 	if err != nil {
-		common.Throw(common.GetRuntime(ctx), err)
+		common.Throw(rt, err)
 	}
-	return bytes
+	ab := rt.NewArrayBuffer(bytes)
+	return &ab
 }
 
 func (c *Crypto) Md4(ctx context.Context, input []byte, outputEncoding string) interface{} {
```

Comment:
This and every other shorthand function also needs to take `ArrayBuffer`

Reply:
Resolved in latest push.